packages/api-client/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './api';
+export * from './api/index';
```

Comment:
This reexport from the index submodule has to be here to get the ApiClientFactory since exporting from ./api doesn't expose it